fairroot/base/sim/FairModule.h

```diff
@@ -147,6 +147,7 @@ class FairModule : public TNamed
      * \deprecated Not used anywhere, will be removed in a future release
      */
     [[deprecated]] FairVolume* getFairVolume(FairGeoNode* fNode);
+    /** Takes ownership of passed volume **/
     void AddSensitiveVolume(TGeoVolume* v);
```

Comment:
Are you actually sure this takes the ownership? Reading your much refactored version lets me doubt this. I could even image, that you could turn this into a `const TGeoVolume*`?

Reply:
You are absolutely right. It creates a new object that gets owned by the `vList`. Will remove the comment.